build/build-pack.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+tmp_dir=$(mktemp -d -t lh-XXXXXXXXXX)
+lh_dir=$(pwd)
```

Comment:
use the `DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"` method to be surefire?